comp/core/tagger/taggerimpl/collectors/workloadmeta_extract.go

```diff
@@ -122,6 +123,8 @@ var (
 	highCardOrchestratorLabels = map[string]string{
 		"io.rancher.container.name": "rancher_container",
 	}
+
+	handledKubernetesMetadataResources = []string{"namespaces", "nodes"}
```

Comment:
question:

how about other types that are explicitly configured for metadata collection by the user? Should we include them here too?

Reply:
Not sure yet, because we don't have any generic option for labels or annotations as tags. The current ones that we have are for pods and annotations only.